ynabber.go

```diff
@@ -105,7 +107,7 @@ func (y *Ynabber) Run() {
 				batch, err := reader.Bulk()
 				if err != nil {
 					y.logger.Error("reading", "error", err, "reader", reader)
-					continue
+					os.Exit(1)
```

Comment:
Will it work for you if we only make the program exit here (nit: use log.Fatal) if interval is set to 0? 

The rate limiting is an intermediate error and will most often disappear at the next run. While we could handle it better (even distinguish between transient errors and permanent ones) its not needed to exit if we are rate limited. As an example i often restart the program for config refreshes which will, with this code change, cause the program to exit immediately.

Reply:
Hmm, did you try it?

My understanding is that unless you exlplicitly set the new config option I added, the loop just continues, see here:
https://github.com/martinohansen/ynabber/blob/d42d225ceb378a147072e54e6db4fba1c064a7d5/reader/nordigen/nordigen.go#L105-L108